spec/abstractops.html

```diff
@@ -57,47 +57,19 @@ <h1>GetOption ( _options_, _property_, _type_, _values_, _fallback_ )</h1>
       1. Assert: Type(_options_) is Object.
       1. Let _value_ be ? Get(_options_, _property_).
       1. If _value_ is *undefined*, return _fallback_.
-      1. Assert: _type_ is *"boolean"* or *"string"*.
+      1. Assert: _type_ is *"boolean"*, *"string"*, *"number"*, or *"stringOrNumber"*.
```

Comment:
rather than `stringOrNumber`, why not accept a List of valid types? I think we do this in a couple places already.

Reply:
Indeed, as in CreateListFromArrayLike? I thought about that, but here `stringOrNumber` is not just allowing either a string or a number, but has to be specific about calling ToString if the value is neither a number or a string.

Maybe having the List of valid types be in order of priority would work? I'll check if that makes things clearer / more general or not.

Reply:
hm, it's a bit tricky if the valid type translates to a coercion algorithm - having `string` or `number` implicitly map to `ToString` or `ToNumber` is awkward, and i definitely see why you'd have gone with `stringOrNumber`.

Reply:
I've done this in https://github.com/tc39/proposal-temporal/pull/1505/commits/ff399a30e119f40c2c978f0952641061bcd5bdc5 but kept it separate for now, I don't have a strong opinion on one being better than the other.

Reply:
Nice, that seems like an improvement to me.